src/renderer/renderer.ts

```diff
@@ -0,0 +1,6 @@
+export default interface Renderer {
+  renderFile(path: string, data: Object)
```

Comment:
I'm still not completely sold on the idea that all implementations of Renderer have to load the files themselves. Ideally this should all be handled in the FileRenderer. In that case FileRenderer won't implement Renderer too, because it's goal is to load files & render them, not render template strings.

Reply:
You're right, this should be part of a FileRenderer interface.

Reply:
Actually, thinking about it some more. If you really wanna go down this road, there should be a `Filesystem` abstraction interface with `LocalFilesystem` implementation.